.github/actions/setup-musl-1_2_5/action.yml

```diff
@@ -0,0 +1,47 @@
+name: Setup musl 1.2.5 toolchain
+description: Install musl 1.2.5 from source and configure the linker for the requested target.
```

Comment:
@dylan-hurd-oai Please update the description to explain this is for DNS fixes with the link from your PR body.